internal/db/psqldb.go

```diff
@@ -0,0 +1,210 @@
+package db
+
+import (
+	"context"
+	"errors"
+
+	"github.com/jackc/pgx/v4"
+)
+
+func CreateTablesAndRoutines(ctx context.Context, db *pgx.Conn) (bool, error) {
+
+	var create_tables string
```

Comment:
var-naming: don't use underscores in Go names; var create_tables should be createTables
Same for `var report_func`

Reply:
Updated in the latest commit.